pkg/controller/controller.go

```diff
@@ -102,8 +105,28 @@ func NewController(clientset kubernetes.Interface, ssclientset ssclientset.Inter
 		},
 	})
 
+	infsecret := informers.NewSharedInformerFactory(clientset, 0).Core().V1().Secrets().Informer()
+	infsecret.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		DeleteFunc: func(obj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(obj)
+			if err == nil {
+				ns, name, err := cache.SplitMetaNamespaceKey(key)
+				if err == nil {
+					ssecret, err := ssclientset.BitnamiV1alpha1().SealedSecrets(ns).Get(context.Background(), name, metav1.GetOptions{})
+					if err == nil {
+						key, err := cache.MetaNamespaceKeyFunc(ssecret)
+						if err == nil {
+							queue.Add(key)
+						}
+					}
+				}
+			}
+		},
+	})
+
 	return &Controller{
 		informer:    informer,
+		infsecret:   infsecret,
```

Comment:
Not a fan of this name. Perhaps it would be better to have `ssInformer` for the SealedSecrets informer and `sInformer` for the Secret informer?

Reply:
Let me check the impact of changing the two of them. Of course, I am going to modify the new one, I like your proposal.